plugins/inputs/statsd/statsd_test.go

```diff
@@ -476,6 +475,7 @@ func TestParse_Distributions(t *testing.T) {
 	parseMetrics()
 	for key, value := range validMeasurementMap {
 		field := map[string]interface{}{
+			//nolint:unconvert // Conversion may be needed for float64 https://github.com/mdempsky/unconvert/issues/40
 			"value": float64(value),
```

Comment:
I also don't get why the conversion is required here. `validMeasurementMap` is a `map[string]float64` so shouldn't `value` be `float64` already?

Reply:
This is because conversion from float to float is sometimes used to prevent the compiler's Fused Multiply Add (FMA) instruction selection to avoid rounding errors. I saw places in Telegraf's code after I removed `float64(float64Var)` conversion tests started to fail.

More details: 
https://stackoverflow.com/questions/63544888/delete-conversion-changes-semantics

It seems that in this case removing conversion works.